cupy/testing/time.py

```diff
@@ -0,0 +1,73 @@
+import time
+
+import numpy
+
+import cupy
+
+
+# Borrowed from https://github.com/niboshi/cupy-perf.
+
+class PerfCaseResult(object):
+    def __init__(self, name, ts):
+        self.name = name
+        self.ts = ts
+
+    def cpu_min(self):
+        return self.ts[0].min()
+
+    def cpu_mean(self):
+        return self.ts[0].mean()
+
+    def cpu_std(self):
+        return self.ts[0].std()
+
+    def gpu_min(self):
+        return self.ts[1].min()
+
+    def gpu_mean(self):
+        return self.ts[1].mean()
+
+    def gpu_std(self):
+        return self.ts[1].std()
```

Comment:
Can you also add `*pu_max()`? I think it'd be useful.

Reply:
Fixed to output max time.

Reply:
Thanks!